x-pack/plugins/gis/public/shared/layers/tile_layer.js

```diff
@@ -13,6 +13,7 @@ import { TileStyle } from '../layers/styles/tile_style';
 export class TileLayer extends AbstractLayer {
 
   static type = "TILE";
+  TMS_LOAD_TIMEOUT = 32000;
```

Comment:
right now `TMS_LOAD_TIMEOUT` on `this` and is editable. Move this out of class and make const